dev/Generated/AnimatedVisualPlayer.properties.cpp

```diff
@@ -0,0 +1,230 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+// DO NOT EDIT! This file was generated by CustomTasks.DependencyPropertyCodeGen
+#include "pch.h"
+#include "common.h"
+#include "AnimatedVisualPlayer.h"
+
+CppWinRTActivatableClassWithDPFactory(AnimatedVisualPlayer)
+
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_AutoPlayProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_DiagnosticsProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_DurationProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_FallbackContentProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_IsAnimatedVisualLoadedProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_IsPlayingProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_PlaybackRateProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_SourceProperty{ nullptr };
+GlobalDependencyProperty AnimatedVisualPlayerProperties::s_StretchProperty{ nullptr };
+
+AnimatedVisualPlayerProperties::AnimatedVisualPlayerProperties()
+{
+    EnsureProperties();
+}
+
+void AnimatedVisualPlayerProperties::EnsureProperties()
```

Comment:
double check and see if all properties have values for working consistent to those in old code.

Reply:
AutoPlay should be true in default. PlayBackRate should be 1 in default. Stretch should be ::Uniform in default. use MUX_DEFAULT_VALUE in AnimatedVisualPlayer.idl to change default value for each conflicted property value.